### PR TITLE
[DEV-225] Profile Page Component

### PR DESCRIPTION
**Pull request by Texian**
[TICKET](https://artistguide.atlassian.net/browse/TICKET)

**Description:**

Description here

Long overdue update and push to dev225 assignment. Required some additional rebasing, but should be fully functional as a skeleton and, hopefully, easy to work with to implement desired final functionality.

**Screenshots (if applicable):**

Screenshots here

**PR Checklist:**

- [ √] If you haven't made changes to `package.json`, `package-lock.json` is not present in this PR
- [ √] You have tested these changes locally by running the app and letting it build successfully
- [ √] Eslint ran successfully upon commit and `--force` was not used to force this PR

_Make sure you **squash and merge** rather than simply merging all commits at once_.

At least **1** PR approval is required by a Tech Lead to be merged. **2** approvals total are required.

### CODE REVIEW

**Comment by bestfoodalex** [comment]
@Texian it looks like this rebasing may have gotten pretty wonky and a bunch of previous changes already in master have ended up in here as new changes. You may want to make a new branch from master, then manually add your profile component changes there. I'd do it for you but it's hard to tell the difference between all of your new code and all of the old code and I don't want to miss anything

**Comment by Texian** [comment]
Ok, I went through and corrected all the merge conflicts, so the branch should be able to merge into the main branch without any issues. *knocks wood*

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# For most projects, this workflow file will not need changing; you simply need
```
Comment (bestfoodalex) [inline .github/workflows/codeql-analysis.yml:1]: Don't include this file

File: src/components/SignUp/ActorInfo2.tsx [hunk 2]
```diff
@@ -232,34 +232,14 @@ const ActorInfo2: React.FC<{
...
-                  ))}
-                </Form.Group>
-              </Col>
-              <Col lg="6">
-                <Form.Group>
-                  <CAGLabelSmaller>
-                    I would be comfortable playing a character through all
-                    phases of their transition: <PrivateLabel />
-                  </CAGLabelSmaller>
+          <Row>
```
Comment (bestfoodalex) [inline src/components/SignUp/ActorInfo2.tsx:235]: You still have a ton of files in here undoing a lot of recent commits - you might want to go through each of these files and `git checkout master [filename]` just so we can make sure we're only keeping your new components and changes to the Profile page

**Pull request closed without merging**